qutebrowser/mainwindow/prompt.py

```diff
@@ -712,6 +728,36 @@ def _init_fileview(self):
         self._file_model.directoryLoaded.connect(
             lambda: self._file_model.sort(0))
 
+    def _get_dirs(self):
+        dirs: Dict[str, List[str]] = {"valid": [], "invalid": []}
+        try:
+            num_rows = self._file_model.rowCount(self._root_index)
+            for row in range(num_rows):
+                index = self._file_model.index(row, 0, self._root_index)
+                hidden = self._to_complete not in index.data()
+                if not hidden:
+                    dirs['valid'].append(index.data())
+                else:
+                    dirs['invalid'].append(index.data())
+        except FileNotFoundError:
+            log.prompt.debug("Directory doesn't exist, can't \
+                             get valid dirs")
+
+        return dirs
+
+    def _do_completion(self, idx, which):
```

Comment:
Is there a reason you moved this up from where the function was originally below? If there is, feel free to keep it here - but if not, I'd prefer moving it down again, to make the diff simpler to review.

Reply:
Yeah feel free to move the function back down.

Reply:
I'm asking you to do so. If we both start pushing changes to your PR, that's just going to result in merge conflicts.